CONTRIBUTING.md

```diff
@@ -157,13 +157,44 @@ cargo upgrade --workspace --to-lockfile
 Incompatible version updates should be done manually after carefully reviewing the changes.
 However you can still use the `cargo upgrade` tool to find out which dependencies could be updated.
 
-```
+```sh
 cargo upgrade --workspace --dry-run
 ```
 
 Then based on the reported potential updates you should manually go and check out what has changed,
 plus how and if it makes sense to update to the newer version.
 
+### Screenshots
+
+We need to update the screenshots. This involves:
+
+- Taking new screenshots.
+- Adding them to the repository in the `screenshots` branch on Github.
+- Updating the links in the README to point to the new screenshots.
+
+### Publishing
+
+Once all the docs, cargo files and screenshots have been updated, we need to
+publish the crate to crates.io.
+
+```sh
+cargo publish --manifest-path="druid-derive/Cargo.toml" --no-verify
+cargo publish --manifest-path="druid-shell/Cargo.toml"
+cargo publish --manifest-path="druid/Cargo.toml"
+```
+
+We need to run the commands in this order, because the druid crate depends on the two other crates. We use `--no-verify` for `druid-derive`, because the crate has a cyclic dependency on `druid` that might cause problem, since we haven't published the latest version of `druid` yet.
+
+<!--- TODO: check that the above is actually true. I'm not sure we really need `--no-verify` --->
```

Comment:
I think we do, since 'derive' depends on the version of 'druid' that we have not yet published.

Reply:
The next person who publishes a new version needs to check.

I'm not sure you need `--no-verify`; because druid is only a dev-dependency, and I'm not sure the crate registry actually checks those.